app/controllers/merchants/products_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Merchants
+  class ProductsController < ApplicationController
+    before_action :require_valid_merchant!, only: [:index]
+
+    def index
+      @products=Product.all
```

Comment:
merchant powinien widzieć tylko swoje produkty

Reply:
aha...bo w tasku z Trello jeśli chodzi o różnice widoczności była mowa o linkach do product details, nie o wszystkich produktach. tzn że wszyscy widzą wszystko, natomiast customerzy widzą linki do szczegółów wszystkich produktów, a merchanci tylko linki do szczegółów swoich produktów. przynajmniej ja tak to zrozumiałam ;P dlatego tak zakodowałam